config/locales/docs/sk.yml

```diff
@@ -1598,16 +1598,20 @@ sk:
               description: Identifikátor schránky
             name:
               description: Názov schránky inštitúcie
-            street:
```

Comment:
Toto je potencialne breaking change ale asi nam to nevadi.

Reply:
To sa zmazalo uz davno, je to uvedene aj v zmenach. Len tu to zostalo (zabudnute), tak som to rovno vycistila.
![image](https://github.com/slovensko-digital/www.ecosystem/assets/43437253/b39777d7-7b6c-460a-8c0d-6f2565629ee2)